package.json

```diff
@@ -21,7 +21,10 @@
   },
   "homepage": "https://github.com/silvermine/eslint-config-silvermine#readme",
   "dependencies": {
-    "@silvermine/eslint-plugin-silvermine": "1.2.1"
+    "@silvermine/eslint-plugin-silvermine": "1.2.1",
+    "eslint-plugin-typescript": "0.12.0",
+    "typescript": "3.0.3",
```

Comment:
Should the TypeScript version be coming from whatever project is using `eslint-config-silvermine` rather than `eslint-config-silvermine` itself? Defining it here, we have the potential of using one version of TypeScript in a project and parsing the TS for linting with a different version.

Also, the latest version is `3.1.6`

Reply:
The problem with removing this is that `typescript-eslint-parser` has a peer dependency on `typescript`. So, if we remove it, we get npm warnings that it's not there. 

I could move it into the `devDependency` section here but then we have the similar craziness that [useragent has](https://github.com/3rd-Eden/useragent/blob/master/index.js#L390-L393).  

The other option is making it a `peerDependency`, but then any JavaScript only project that we have will need to include `typescript` or get `peerDependency` warnings.

Reply:
Ok, thanks for the explanation. I now also see the reason for having the version be `3.0.3`: https://github.com/eslint/typescript-eslint-parser#supported-typescript-version

So this means that repos that use this eslint config will be held back to whatever version of TypeScript that `typescript-eslint-parser` supports.